src/server/api/notification.ts

```diff
@@ -4,14 +4,15 @@ import {notifyLaravel} from "../utils";
 const router = express.Router();
 
 router.post('/', (req, res) => {
-    const {title, body} = req.body
+    const {title, body, customEvent: event} = req.body
```

Comment:
```suggestion
    const {title, body, event: customEvent} = req.body
```

Reply:
Did you mean to also submit suggestions for line 8? And I'm not actually sure what the response from the API is for retrieving the event name provided when registering a notification.